src/Compilers/VisualBasic/Portable/Semantics/Conversions.vb

```diff
@@ -234,6 +235,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Operator
 
         ''' <summary>
+        ''' Creates a <seealso cref="CommonConversion"/> from this Visual Basic conversion.
+        ''' </summary>
+        ''' <returns>The <see cref="CommonConversion"/> that represents this conversion.</returns>
+        ''' <remarks>
+        ''' This is a lossy conversion; it is not possible to recover the original <see cref="Conversion"/>
+        ''' from the <see cref="CommonConversion"/> struct.
+        ''' </remarks>
+        Public Function ToCommonConversion() As CommonConversion
+            Return New CommonConversion(Exists, IsIdentity, IsNumeric, IsReference, MethodSymbol)
```

Comment:
Yes, because C# has both MethodGroup conversions and user-defined conversions. There are only user-defined conversions here, so no need to check the condition.